### PR TITLE
[NA-245] - remove UserGet Schema base_token field

### PR DESCRIPTION
**Pull request by bjornaer**
commented out base token while checking if it would be a breaking change

also commented test cases that broke due to commented function in codebase

### CODE REVIEW

**Comment by alexpearce** [comment]
What's the commented-out function? Do you think it's worth re-adding, given we have to drop several tests? Maybe @paulcjh can comment on whether the code should be uncommented.

**Comment by bjornaer** [comment]
> What's the commented-out function? Do you think it's worth re-adding, given we have to drop several tests? Maybe @paulcjh can comment on whether the code should be uncommented.

I have not done so because I lack context as to why it was commented in first place -- so i adjusted tests to just stop breaking

**Comment by paulcjh** [comment]
> > What's the commented-out function? Do you think it's worth re-adding, given we have to drop several tests? Maybe @paulcjh can comment on whether the code should be uncommented.
> 
> I have not done so because I lack context as to why it was commented in first place -- so i adjusted tests to just stop breaking

The tests are super out of date, I've had to omit them from my personal workflow for the moment because of how frequently I've been changing the schemas. For the moment don't worry about them. We'll sort this out in a future PR.

File: pipeline/schemas/user.py [hunk 1]
```diff
@@ -18,12 +18,15 @@ class UserBase(AvatarHolder):
...
 class UserGet(UserBase):
     id: str
-    base_token: TokenGet
     oauth_provider: Optional[str]
     verified: Optional[bool]
     subscribed: Optional[bool]
 
 
+class EnrichedUserGet(UserGet):
+    base_token: TokenGet
```
Comment (alexpearce) [inline pipeline/schemas/user.py:27]: Unfortunately the [naming conventions](https://github.com/neuro-ai-dev/pipeline_schemas#naming-conventions) didn't survive the schema migration but `UserGetEnriched` would be more consistent with other 'extended' names. I wouldn't be against something more explicit here e.g. `UserGetWithBaseToken`.

**Review by alexpearce — APPROVED** [review]
LGTM, thank you! Just a minor suggestion from me for your consideration.

File: pipeline/schemas/user.py (hunk 1 above)
Comment (bjornaer) [reply pipeline/schemas/user.py:27]: I thought about making it specific, but then erinched felt it delivers the idea and allows for more data to be added in the future, I switch around the wording!

**Pull request merged**